app/main.py

```diff
@@ -1,6 +1,7 @@
+import os
 from gensim.models import KeyedVectors
 
-GLOVE_PATH = 'glove.twitter.27B.25d.word2vec.p'
+GLOVE_PATH = '{}/glove.twitter.27B.25d.word2vec.p'.format(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
@jkarimi91 I made this change. Lets always use absolute paths when loading any files. From this link: https://stackoverflow.com/questions/5137497/find-current-directory-and-files-directory

Reply:
@bvallelunga I refactored it to be more pythonic as well as robust (by using `os.path.join`). This PR is now ready for you to review again.